packages/frontend/src/features/catalog/components/CatalogPanel.tsx

```diff
@@ -69,112 +84,217 @@ export const CatalogPanel: FC<React.PropsWithChildren<Props>> = ({
+                                                    }}
+                                                    catalogItem={item}
+                                                    searchString={
+                                                        debouncedSearch
+                                                    }
+                                                    isSelected={
+                                                        metadata !==
+                                                            undefined &&
+                                                        item.name ===
+                                                            selectedTable
+                                                    }
+                                                    tableUrl={`/projects/${projectUuid}/tables/${item.name}`}
+                                                />
+                                            ))}
+                                    </tbody>
+                                </Table>
+                            </CatalogGroup>
+                        ))}
 
-                <Table h="100%">
-                    <tbody>
-                        {ungroupedCatalogItems
-                            .sort((a, b) => a.name.localeCompare(b.name))
-                            .map((item, idx) => (
+                    <Table h="100%">
+                        <tbody>
+                            {ungroupedCatalogItems.map((item, idx) => (
                                 <CatalogListItem
                                     key={`${item.name}-${idx}`}
                                     catalogItem={item}
                                     searchString={debouncedSearch}
                                     tableUrl={`/projects/${projectUuid}/tables/${item.name}`}
+                                    isSelected={
+                                        metadata !== undefined &&
+                                        item.name === selectedTable
+                                    }
+                                    onClick={() => {
+                                        selectAndGetMetadata(item.name);
+                                    }}
                                 />
                             ))}
-                    </tbody>
-                </Table>
+                        </tbody>
+                    </Table>
+                </Stack>
+            </Stack>
+            <Stack>
+                <Button
+                    onClick={() => {
+                        if (metadata) closeMetadata();
+                        else if (selectedTable === undefined)
+                            selectAndGetMetadata(
+                                ungroupedCatalogItems[0]?.name,
+                            );
+                        else selectAndGetMetadata(selectedTable);
+                    }}
+                >
+                    {metadata ? 'Show metadata' : 'Hide metadata'}
```

Comment:
This is backwards. It says show when you can hide. But, this is also obviously very temporary. I can fix this in the next review if you want.